deploy/frontend.yml

```diff
@@ -18,32 +21,97 @@ objects:
         paths:
           - /apps/patch
       image: ${IMAGE}:${IMAGE_TAG}
-      navItems:
-        - appId: "patch"
-          title: "Patch"
-          href: "/insights/patch"
-          product: "Red Hat Insights"
-          expandable: true
-          routes:
-            - appId: 'patch'
-              title: 'Advisories'
-              href: '/insights/patch/advisories'
-            - appId: 'patch'
-              title: 'Systems'
-              href: '/insights/patch/systems'
-            - appId: 'patch'
-              title: 'Packages'
-              href: '/insights/patch/packages'
-            - appId: 'patch'
-              title: 'Templates'
-              href: '/insights/patch/templates'
       module:
-        manifestLocation: "/apps/patch/fed-mods.json"
+        manifestLocation: /apps/patch/fed-mods.json
         modules:
-          - id: "patch"
-            module: "./RootApp"
+          - id: patch
+            module: ./RootApp
             routes:
               - pathname: /insights/patch
+      navigationSegments:
+        - segmentId: patch-advisories-packages
+          navItems:
+            - id: advisories
+              title: Advisories
+              href: /insights/patch/advisories
+            - id: packages
+              title: Packages
+              href: /insights/patch/packages
+        - segmentId: patch-systems
+          navItems:
+            - id: patchSystems
+              title: Systems
+              href: /insights/patch/systems
```

Comment:
I think instead of `navigationSegments` block we need to have this:
```
      bundleSegments:
        - segmentId: content
          bundleId: insights
          position: 500
          navItems:
            - id: content
              title: Content
              expandable: true
              routes:
                - id: advisories
                  title: Advisories
                  href: /insights/patch/advisories
                - id: packages
                  title: Packages
                  href: /insights/patch/packages
                - id: patchSystems
                  title: Systems
                  href: /insights/patch/systems
```